### ISSUE TITLE
Support Citus 11

### ISSUE THREAD

**Issue by nathanieltalbot** [rapidloop/pgmetrics#54]
Citus 11.x introduced a couple of breaking changes to the tables that pgmetrics queries, which causes errors when pgmetrics is run against Citus clusters on Citus 11. See release notes for details: https://www.citusdata.com/updates/v11-0/. The specific breaking changes are:

`citus_worker_stat_activity` has been dropped in favor of `citus_stat_activity` and `citus_dist_stat_activity`. https://github.com/citusdata/citus/pull/5760. This causes this error when querying:
```
pgmetrics: warning: citus_worker_stat_activity query failed: pq: relation "citus_worker_stat_activity" does not exist
```

`citus_dist_stat_activity` does not have `query_hostname` and is a view over `citus_stat_activity`: https://github.com/citusdata/citus/pull/5760 which causes an issue when trying to get `query_hostname`:
```
pgmetrics: warning: citus_dist_stat_activity query failed: pq: column "query_hostname" does not exist
```

`citus_lock_waits` no longer has column `waiting_pid` but `waiting_gpid` (see: https://github.com/citusdata/citus/pull/5766) which causes this issue when trying to query the table:
```
pgmetrics: warning: citus_lock_waits query failed: pq: column "waiting_pid" does not exist
```

Would it be possible to get these models updated for the most recent major version of Citus?

**Comment by mdevan** [comment]
Thanks for listing out the breaking changes.

The next release of pgmetrics will have support for Citus 11.

**Comment by mdevan** [comment]
Available in release v1.14.0.
